models/consistency.go

```diff
@@ -9,6 +9,7 @@ import (
 	"strings"
 
 	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/models/unittest"
```

Comment:
I worry about that the `testing` and `assert` packages come into non-unit-test code again.

If we want to keep the unit test code away from non-unit-test code, the steps might be:

1. refactor the `checkConsistency*` functions first, move them to a unit test package.
2. move `models/db/unit_test.go` to `models/unittest/`
3. drop the `unittestbridge`